config/application.rb

```diff
@@ -6,7 +6,7 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
-Dotenv::Railtie.load
+Dotenv::Railtie.load if %w(development test).include?(Rails.env)
```

Comment:
I made this change so that Dotenv would not load on `RAILS_ENV=production`. When environment is production, the Heroku configuration variables will be used.